core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -189,6 +191,67 @@ oppia.controller('CreatorDashboard', [
       return UrlInterpolationService.getStaticImageUrl(iconUrl);
     };
 
+    $scope.showCreateQuestionModal = function() {
+      var question = QuestionObjectFactory.createDefaultQuestion();
+      TopicsAndSkillsDashboardBackendApiService.fetchDashboardData().then(
+        function(response) {
+          var topicSummaries = response.data.topic_summary_dicts;
+          $uibModal.open({
+            templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+              '/pages/creator_dashboard/create_question_modal_directive.html'),
+            backdrop: true,
+            size: 'lg',
+            resolve: {},
+            controller: [
+              '$scope', '$uibModalInstance', function(
+                  $scope, $uibModalInstance) {
+                $scope.question = question;
+                $scope.topicId = null;
+                $scope.questionStateData = $scope.question.getStateData();
+                $scope.topicSummaries = topicSummaries;
+
+                $scope.isNotValidQuestion = function() {
+                  return !$scope.question.validate();
+                };
+
+                $scope.dismissModal = function() {
+                  $uibModalInstance.dismiss();
+                };
+
+                $scope.createQuestion = function() {
+                  $uibModalInstance.close({
+                    question: question,
+                    topicId: $scope.topicId
+                  });
+                };
+              }
+            ]
+          }).result.then(function(result) {
+            var topicVersion = null;
+            for (var i = 0; i < topicSummaries.length; i++) {
+              if (topicSummaries[i].id === result.topicId) {
+                topicVersion = topicSummaries[i].version;
+                break;
+              }
+            }
+            $http.post('/generalsuggestionhandler/', {
+              suggestion_type: 'add_question',
+              target_type: 'topic',
+              target_id: result.topicId,
+              target_version_at_submission: topicVersion,
+              change: {
+                cmd: 'create_new_fully_specified_question',
+                question_dict: result.question.toBackendDict(true),
```

Comment:
@aks681 -- I looked at this in the codebase and it looks like the schemaVersion arg in toBackendDict() is unused. Can we remove it?

Reply:
@aks681 ping!